server.js

```diff
@@ -10,38 +1,47 @@
+const express = require("express");
+var app = express();
+var server = require("http").Server(app);
+var io = require("socket.io")(server);
+const port = 6969;
+
+app.get("/", (req, res) => {
+  res.sendFile("templates/client_chat.html", { root: __dirname });
+});
 
-app.get('/darkmode.css', (req, res) => {
-    res.sendFile('darkmode.css', { root: __dirname });
-})
+app.get("/darkmode.css", (req, res) => {
+  res.sendFile("darkmode.css", { root: __dirname });
+});
 
-app.get('/manifest.webmanifest', (req, res) => {
-    res.sendFile('manifest.webmanifest', { root: __dirname });
-})
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/CryptoChat-dev/cryptochat/pullRequest?prid=7237627)
```suggestion
app.get('/manifest.webmanifest', (req, res) => {
```